### ISSUE TITLE
Reduced hemoglobin A1c

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5954]
**Preferred term label:** Reduced hemoglobin A1c

**Synonyms** Reduced glycosylated hemoglobin; Decreased glycated hemoglobin, Decreased HbA1c levels

**Definition (free text, please give PubMed ID)** A reduced concentration of hemoglobin A1c (HbA1c), which is the product of nonenzymatic attachment of a hexose molecule to the N-terminal amino acid of the hemoglobin molecule.

**Parent term (use hpo.jax.org/app)** Abnormal hemoglobin HP:0011902

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4734630/

**Comment by pnrobinson** [comment]
I do not see clear lower boundaries below which a lab value would be considered abnormal in an individual. I think that studies on low HbA1c have been conducted in cohorts.
